docs/UserGuide.md

```diff
@@ -384,6 +400,13 @@ If your changes to the data file makes its format invalid, AddressBook will disc
 Furthermore, certain edits can cause the AddressBook to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
 </div>
 
+---
+
+## Future Features
+
+1) While we are aware of the possibility of international numbers, for this iteration, we decided to focus on the local population.
```

Comment:
Could you explicitly state support for international numbers is planned? E.g. "Support for these numbers is planned in a future release." I know the current statement implies it but I think better to state it clearly.